### PR TITLE
kubernetes fileeats

### PR DESCRIPTION
**Pull request by thecraftman**
ingest data into Axiom using K8s

### CODE REVIEW

File: docs/data-shippers/kubernetes-filebeat.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
+<div class="axi-header">
+  <h1>Ingest using Kubernetes</h1>
+</div>
+
+You can ingest data into Axiom by configuring the **Kubernetes** `.yaml` file
```
Comment (mhr3) [inline docs/data-shippers/kubernetes-filebeat.md]: Sounds a bit weird, how about:

You can ingest logs from your Kubernetes cluster into Axiom using filebeat.

File: docs/data-shippers/kubernetes-filebeat.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
+<div class="axi-header">
+  <h1>Ingest using Kubernetes</h1>
+</div>
+
+You can ingest data into Axiom by configuring the **Kubernetes** `.yaml` file 
+
+The following is an example of a Deployment configuration to ingest your data logs into **Axiom** using Kubernetes.
```
Comment (mhr3) [inline docs/data-shippers/kubernetes-filebeat.md]: it's a DaemonSet, not a Deployment

File: docs/data-shippers/kubernetes-filebeat.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+              value: http://axiom:80
+            - name: AXIOM_DATASET_NAME
+              value: dataset
+            - name: AXIOM_INGEST_TOKEN
+              value: xait-xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+```
+
+where:
+
+- **AXIOm_HOST** is your Axiom deployment URL.
```
Comment (mhr3) [inline docs/data-shippers/kubernetes-filebeat.md]: lowercase `m`

File: docs/data-shippers/kubernetes-filebeat.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+
+You can ingest data into Axiom by configuring the **Kubernetes** `.yaml` file 
+
+The following is an example of a Deployment configuration to ingest your data logs into **Axiom** using Kubernetes. 
+
+
+## Configuration
+
+
+```yaml
```
Comment (mhr3) [inline docs/data-shippers/kubernetes-filebeat.md:12]: There's nothing mentioning what to do with this yaml, please add the command

Comment (thecraftman) [reply docs/data-shippers/kubernetes-filebeat.md:12]: @mhr3 do you mean the command to run the yaml file?

Comment (mhr3) [reply docs/data-shippers/kubernetes-filebeat.md:12]: yes

**Pull request merged**